### PR TITLE
adds nc-news-8

### PR DESCRIPTION
**Pull request by RichRavenM**

### CODE REVIEW

File: models/articles.models.js [hunk 1]
```diff
@@ -41,6 +41,28 @@ exports.selectCommentsByArticleId = (article_id) => {
...
+          status: 404,
+          msg: "Article id does not exist",
+        });
+      }
+      if (rows[0].votes < 0) {
+        return Promise.reject({
+          status: 400,
+          msg: "Cannot set votes to be a negative number",
+        });
+      }
```
Comment (Dutton1990) [inline models/articles.models.js]: This should happen before the query, you will still be setting the votes to be a negative number through the query even though you return an error

**Review by Dutton1990 — APPROVED** [review]
Great job Rich, your extra test on votes is not something we asked for but this is your server and you can deal with negative numbers that way if you like 😄 

I have left a suggestion on that one though and you should handle it differently. 

Otherwise you've handled everything we are looking for, ace job! Merge when you please
